app/containers/Questions/Content/ShowMoreButton.jsx

```diff
@@ -19,19 +19,26 @@ const cookieFilterSetter = value => ({
   },
 });
 
-export const ShowMoreButton = ({ changeQuestionFilterDispatch, children, questionFilterFromCookies }) => {
+export const ShowMoreButton = ({
+  changeQuestionFilterDispatch,
+  children,
+  questionFilterFromCookies,
+}) => {
   const [filter, setFilterValue] = useState(+(questionFilterFromCookies || 1));
 
-  useEffect(() => {
-    const cookieValue = questionFilterFromCookies;
+  useEffect(
+    () => {
+      const cookieValue = questionFilterFromCookies;
 
-    setFilterValue(cookieValue);
+      setFilterValue(cookieValue);
 
-    if (!cookieValue || cookieValue === '1') {
-      // setCookie(cookieFilterSetter(1));
-      // changeQuestionFilterDispatch(1);
-    }
-  }, [filter]);
+      if (!cookieValue || cookieValue === '1') {
+        // setCookie(cookieFilterSetter(1));
+        // changeQuestionFilterDispatch(1);
+      }
+    },
+    [filter],
```

Comment:
do we need to include `questionFilterFromCookies` to dependency array of useEffect?